packages/typescript/src/index.ts

```diff
@@ -17,6 +17,8 @@ import { preflight } from './preflight';
 import createWatchProgram, { WatchProgramHelper } from './watchProgram';
 import TSCache from './tscache';
 
+const defaultTsExtensionsGlob = '?(m|c)ts+(|x)';
```

Comment:
It would be great to have a doc comment here to explain what patterns this glob is matching, so future contributors know :)

Reply:
added